research/picai/central/launch.slrm

```diff
@@ -55,4 +54,25 @@ source ${VENV_PATH}bin/activate
 echo "Active Environment:"
 which python
 
-python ~/FL4Health/research/picai/central/train.py --checkpoint_dir ${LOG_DIR} --fold ${FOLD_ID} --run_name ${RUN_NAME} > ${LOG_PATH} 2>&1
+# Function that is called to requeue current slurm job
+handler()
+{
+	echo "Requeue $SLURM_JOB_ID at $(date)"
```

Comment:
Does `$(date)` also contain a timestamp (i.e. not just the day, month, year)? If not, perhaps we want that granularity, if so, please ignore 🙂

Reply:
It is of the form: `Thu May 30 09:41:36 EDT 2024`. So  includes a time :)